addons/payment/models/payment_acquirer.py

```diff
@@ -204,6 +211,31 @@ def _inverse_journal_id(self):
             elif payment_method_line:
                 payment_method_line.unlink()
 
+    def _compute_support_authorization(self):
+        """
+        For an acquirer to support the authorization feature, it must override this method.
+        """
+        self.write({'support_authorization': False})
```

Comment:
inside a compute you do not use write:
self.support_authorization = False

no ?

Reply:
Hello @fmdl, this is a draft pull request and therefore the code is not final and is subject to change. There is no point in reviewing it now. Thank you! :)